src/Serilog.Sinks.MSSqlServer/Sinks/MSSqlServer/MSSqlServerSinkOptions.cs

```diff
@@ -47,6 +48,11 @@ public MSSqlServerSinkOptions()
         /// </summary>
         public bool AutoCreateSqlTable { get; set; }
 
+        /// <summary>
+        /// Flag to prevent logging SQL commands from taking part in ambient transactions (default: true)
+        /// </summary>
+        public bool PreventEnlistInTransaction { get; set; }
```

Comment:
Multiple times while reviewing this I had to think about the value as this is the opposite of the underlying `Enlist` property.

Did you consider naming it `EnlistInTransaction`, `EnlistTransaction`, `EnlistAmbientTransaction`, `EnlistCurrentTransaction` or `EnlistInDistributedTransaction` (if applicable) instead, with a default of false?

Essentially `Enlist=false` is the base set up and you turn on the feature to enlist (even though this isn't the connection string default, it should be the logging default).

Reply:
I named it that way because I wanted the semantics to be that the sink does not touch the connection string passed by the user if this option is set to false. If the user passes Enlist=True it will stay true if the user passes Enlist=False it will also stay like this. Only if the Prevent... option is set to true the sink will set Enlist=False. But I guess you are right. This would confuse users and also it is a double negation. 

I will change it. Thanks! :)

Reply:
> I wanted the semantics to be that the sink does not touch the connection string passed by the user if this option is set to false

I think that only works if the sink defaulted to enlisting, the default for the underlying SQL connection.